microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -1741,6 +1797,9 @@ components:
       type: number
       minimum: 0
       maximum: 1
+    taxonomyLevel:
```

Comment:
Where would a user obtain the possible values for this?

Reply:
How would you feel about this being an enum?

Reply:
That's probably fine. Enum is annoying if things need to be inferred at run time, but should be fine since taxonomic levels should be consistent over time, right?